text/2018-10-15-batch-client-messages.md

```diff
@@ -0,0 +1,69 @@
+# Summary
+
+Add a new API `BatchCommands` into `Tikv` service, which contains a vector of raw client requests (e.g. `GetRequest`). It can reduce gRPC messages count obviously, and according to tests the performance is improved considerably, especially when TiKV or TiDB's CPU usage is the bottle-neck.
```

Comment:
seem we can use bottleneck directly? no need - here. 

@lilin90

Reply:
@siddontang Yes. "bottleneck" is the standard version for this word.